packages/client/index.js

```diff
@@ -39,6 +39,11 @@ async function makeMove(move) {
   return parent.makeMove(move);
 }
 
+async function now() {
+  const parent = await connection.promise;
```

Comment:
Is there a way to make this function asynchronous? That is the expectation from `semantle-battle perspective`

Reply:
I'm thinking that we can default now to use the local Data.now() until the promise is ready

Reply:
done

Reply:
actually not done cant do bc penpal converts all to async anyway